src/connections/allContacts.js

```diff
@@ -104,3 +102,18 @@ export const withContactsMutations = withMutations(client => ({
 }))
 
 export default flow([withContacts, withContactsMutations])
+
+export function mergeContact(contact, attributes) {
+  function concat(arr1 = [], arr2 = []) {
+    return [...arr1, ...arr2]
```

Comment:
:ok_hand: It's weird that neither `Array#concat()`, `Array.from()`, `_.concat()` nor the spread operator handle a `null` value as an empty sequence :confused:

Reply:
Ahhh EcmaScript :disappointed:
:rofl: